core/storage/job/gae_models.py

```diff
@@ -94,17 +110,38 @@ def get_recent_jobs(cls, limit, recency_msec):
 
     @classmethod
     def get_all_unfinished_jobs(cls, limit):
+        """Get all jobs that are unfinished.
+
+        Args:
+            limit: int. A limit on the number of jobs to return.
+
+        Returns:
+            A list of jobs. Possiably none.
+        """
         return cls.query().filter(
             JobModel.status_code.IN([STATUS_CODE_QUEUED, STATUS_CODE_STARTED])
         ).order(-cls.time_queued_msec).fetch(limit)
 
     @classmethod
     def get_unfinished_jobs(cls, job_type):
+        """Get jobs that are unfinished.
```

Comment:
Let's keep the tense consistent: always third-person singular. So "gets" instead of "get".